galaxychop/core.py

```diff
@@ -550,8 +499,8 @@ def jcirc(self, bin0=0.05, bin1=0.005):
 
         Jz = kk / np.max(np.abs(kk))
 
-        # Make the energy binning and select the Jz values with which we
-        # calculate the J_circ.
+        # Make the specific energy binning and select the Jz values with
```

Comment:
Build instead of make?
"Build the specific energy binning and select the Jz values to calculate J_circ"